haptic_control:
  ros__parameters:
    tool_link_name: "lbr_link_ee"
    base_link_name: "lbr_link_0"
    ft_link_name: "lbr_link_ee"
    ft_sensor_rate: 1000.0
    ft_topic_name: "/todo"